jumpscale/packages/tfgrid_solutions/chats/kubernetes.py

```diff
@@ -42,23 +45,18 @@ def kubernetes_name(self):
     @chatflow_step(title="Master and Worker flavors")
     def choose_flavor(self):
         form = self.new_form()
-        sizes = ["1 vCPU 2 GiB ram 50GiB disk space", "2 vCPUs 4 GiB ram 100GiB disk space"]
+        sizes = [
+            f"vCPU: {data.get('cru')}, RAM: {data.get('mru')} GiB, Disk Space {data.get('sru')} GiB"
+            for data in self.KUBERNETES_SIZES.values()
+        ]
         cluster_size_string = form.drop_down_choice("Choose the size of your nodes", sizes, default=sizes[0])
-
-        self.workernodes = form.int_ask(
```

Comment:
why remove the number of workers? you can allow 0 nodes for the cluster, and but removing the option entirely to force the user to extend the cluster is not a good ux inmo

Reply:
We checked that with @xmonader  before developing it